drivers_license/config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "category": "drivers_license",
+    "description": "state drivers license",
```

Comment:
for descriptions, let's use grammatically correct English here -- caps, punctuation (for example, "driver's license")